spring-cloud-stream-binder-jms-common/src/main/java/org/springframework/cloud/stream/binder/jms/utils/ListenerContainerFactory.java

```diff
@@ -26,22 +27,27 @@
  * Factory to create Jms ListenerContainer
  *
  * @author José Carlos Valero
+ * @author Donovan Muller
  * @since 1.1
  */
 public class ListenerContainerFactory {
 
+    private JmsBinderConfigurationProperties configurationProperties;
+
     private ConnectionFactory factory;
 
-    public ListenerContainerFactory(ConnectionFactory factory) {
+    public ListenerContainerFactory(JmsBinderConfigurationProperties configurationProperties,
+                                    ConnectionFactory factory) {
+        this.configurationProperties = configurationProperties;
         this.factory = factory;
     }
 
     public AbstractMessageListenerContainer build(Queue group) {
-        AbstractMessageListenerContainer listenerContainer = new SimpleMessageListenerContainer();
+        DefaultMessageListenerContainer listenerContainer = new DefaultMessageListenerContainer();
         listenerContainer.setDestination(group);
         listenerContainer.setPubSubDomain(false);
         listenerContainer.setConnectionFactory(factory);
-        listenerContainer.setSessionTransacted(true); //Maybe configurable?
+        listenerContainer.setSessionTransacted(configurationProperties.isTransacted());
```

Comment:
See comment in properties - I am not sure it's wise to expose this as a property with a DMLC.